pkgs/applications/misc/rofi/wayland.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, rofi-unwrapped
+, wayland-protocols
+, wayland
+}:
+
+rofi-unwrapped.overrideAttrs (oldAttrs: rec {
+  pname = "rofi-wayland-unwrapped";
+  version = "1.7.1+wayland1";
+
+  src = fetchFromGitHub {
+    owner = "lbonn";
+    repo = "rofi";
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "8CLBBRvtz9nYAHJLdBUX99sH3ZC+242wUtE7tXm5B7o=";
+  };
+
+  buildInputs = oldAttrs.buildInputs ++ [ wayland wayland-protocols ];
```

Comment:
wayland-protocols goes to nativeBuildInputs.

Reply:
Thanks, updated!
(and rebased on master, since #150165 was merged)